_account-security/security/encryption/ssh-for-microsoft-azure.md

```diff
@@ -0,0 +1,136 @@
+
+    substeps:
+      - title: "Launch the virtual machine"
+        anchor: "launch-virtual-machine"
+        content: |
+          1. In the sidenav, click **Virtual machines**.
+          2. On the **Virtual machines** page, click the **+ Add** button.
+          3. Fill in the fields in the tabs. In the list below are the fields that require specific values for Stitch:
+             - **Basics tab**:
+                - **Public inbound ports**: Select **Allow selected ports**.
+                - **Select inbound ports**: Select **SSH (22)**.
+          4. After you've finished filling in the required fields, click **Review + create**.
+          5. Click **Create** to create the virutal machine.
+
+          After you click **Create**, Azure will launch the deployment process for the virtual machine. This may take a few minutes to complete.
+
+      - title: "Configure the virtual machine's inbound access"
+        anchor: "configure-virtual-machine-inbound-access"
+        content: |
+          {% capture ip-list %}
+          {% for ip-address in ip-addresses %}{{ ip-address.ip }}{% unless forloop.last == true %},{% endunless %}{% endfor %}
+          {% endcapture %}
+
+          After Azure finishes deploying the virtual machine, you can move onto configuring the access rules for its security group. Inbound access rules will allow traffic from Stitch's IP addresses to access the virtual machine.
+
+          1. In the sidenav, click **Virtual machines**.
+          2. On the **Virtual machines** page, click virtual machine you created in the previous step.
+          3. The details page for the virtual machine will display. In the middle menu, click **Networking**.
+          4. In the **Inbound Port Rules** section, click the **Add inbound port rule** button.
+          5. In the **Add inbound security rule** tab that displays, fill in the fields as follows. If a field isn't in this list, **use the default value**:
+             - **Source**: Select **IP Addresses**.
+             - **Source IP addresses/CIDR ranges**: Paste this comma-delimited list of Stitch's IP addresses:
+
+               ```markdown
+               {{ ip-list | strip }}
+               ```
+             - **Protocol**: Select **TCP**.
+             - **Action**: Select **Allow**.
+             - **Name**: Enter `stitch`.
+          6. When finished, click **Add** to create the inbound rule.
+
+  - title: "Enable the SSH server to access the database"
+    anchor: "enable-ssh-server-access"
+    content: |
+      Next, you'll configure the {{ destination.display_name }} database to allow traffic forwarded from the virtual machine to access the database server. This is accomplished by whitelisting the virtual machine's public IP address in the server's firewall settings.
```

Comment:
If you use the *public IP* then this tunnel isn't really accomplishing anything. This should be private IP.